tests/percy/camera.js

```diff
@@ -0,0 +1,60 @@
+const { SnapshotWidths } = require('./constants');
+
+/**
+ * Responsible for taking Percy snapshots
+ */
+class Camera {
+  /**
+   * @param {Object} obj
+   * @param {function} obj.percySnapshot The percy snapshot function
+   * @param {boolean} obj.iframeMode Enables capturing iframe snapshots
+   */
+  constructor({ percySnapshot, iframeMode }) {
```

Comment:
Since there are only two parameters here, do we need to wrap them in an object?

Reply:
no, I can simplify it